src/App.css

```diff
@@ -5,7 +5,18 @@
   font-style: italic;
   margin: 40px 0 0;
 }
+.dg_popular_star {
```

Comment:
this rule is redundant to starting on line 12, and line 12's is a bit safer.

Reply:
opps good catch. This has been removed.